### PR TITLE
Preserve data if crm_lead_lost_reason is installed

### PR DESCRIPTION
**Pull request by cubells**
crm_lead_lost_reason module features are included in crm module.

These changes preserve module data if user has that module installed.

cc @Tecnativa

### CODE REVIEW

File: addons/crm/migrations/9.0.1.0/pre-migration.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
     ],
     'crm_lead': [
         ('section_id', 'team_id'),
+        ('lost_reason_id', 'lost_reason'),
```
Comment (pedrobaeza) [inline addons/crm/migrations/9.0.1.0/pre-migration.py]: Don't include this here. You are already doing this in the SQL

File: addons/crm/migrations/9.0.1.0/openupgrade_analysis_work.txt [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---Fields in module 'crm'---
```
Comment (pedrobaeza) [inline addons/crm/migrations/9.0.1.0/openupgrade_analysis_work.txt]: Why are you touching this file?

Comment (cubells) [reply addons/crm/migrations/9.0.1.0/openupgrade_analysis_work.txt]: Undone.

Comment (pedrobaeza) [reply addons/crm/migrations/9.0.1.0/openupgrade_analysis_work.txt]: No, changes are still here.

File: addons/crm/migrations/9.0.1.0/pre-migration.py [hunk 3]
```diff
@@ -89,3 +89,5 @@ def migrate(cr, version):
         [('4', '3')], table='crm_lead',
     )
     cr.execute("update crm_lead set type='opportunity' where type is null")
+    cr.execute("ALTER TABLE IF EXISTS crm_lead_lost_reason "
```
Comment (pedrobaeza) [inline addons/crm/migrations/9.0.1.0/pre-migration.py]: Include a comment indicating that this is for migrating module ....

**Review by StefanRijnhart — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**